scripts/ci/pull_translations.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 echo "::group::tx-pull"
-./tx pull --minimum-perc=50 --force
+./tx pull -a --minimum-perc=50 --force
```

Comment:
```suggestion
# Of all the languages that exist on the remote, 
# force-pull only those with at least 50% translation coverage
./tx pull -a --minimum-perc=50 --force
```

Reply:
```suggestion
# `-a` Of all the languages that exist on the remote, 
# `--minimum-perc=50 --force` force-pull only those with at least 50% translation coverage
./tx pull -a --minimum-perc=50 --force
```

I assume the inention is to explain the params